src/Tablet/Dashboard/WeatherTile.js

```diff
@@ -68,7 +72,7 @@ const WeatherTile = () => {
             fontSize: 24,
             marginTop: 5,
             marginBottom: 10,
-            textAlign: "center",
+            textAlign: "right",
```

Comment:
This file was completely refactored and sitting in a PR. It's a good practice to review PRs often to be aware of changes that impact our work 😄